nilearn/region_extraction/regionextractor.py

```diff
@@ -0,0 +1,341 @@
+"""
+region_signal_extractor
+"""
+
+import nibabel
+import numpy as np
+
+from scipy.ndimage import label
+from scipy.stats import scoreatpercentile
+
+from sklearn.base import clone
+from sklearn.externals.joblib import Memory
+
+from skimage.feature import peak_local_max
```

Comment:
The corresponding skimage code will be added to nilearn/external. No additional dependency.